src/config/dynamic-theme-fixes.config

```diff
@@ -17739,6 +17739,22 @@ INVERT
 
 ================================
 
+shipstation.com
+
+INVERT
+.note-icon-3F9ryyv
+.sub-item-icon-1L2L6PE
```

Comment:
```suggestion
div[class*="note-icon"]
div[class*="sub-item-icon"]
```

Reply:
Hi, these replacements don't work to actually revert the targeted icons. When I text it out with your lines instead of mine, the icons I was trying to revert are not affected.